### PR TITLE
Feature ssh lua implementation

### PR DESCRIPTION
**Pull request by Icety**

### CODE REVIEW

File: services/ssh/ssh-simulator.go [hunk 1]
```diff
@@ -395,12 +407,28 @@ func (s *sshSimulatorService) Handle(ctx context.Context, conn net.Conn) error {
 								event.Custom("ssh.command", line),
 							))
 
-							term.Write([]byte(fmt.Sprintf("%s: command not found\n", line)))
+							resp, _ := sConn.Handle(line)
+							term.Write([]byte(resp))
+
+							// term.Write([]byte(fmt.Sprintf("%s: command not found\n", line)))
```
Comment (martijnjanssen) [inline services/ssh/ssh-simulator.go]: If you're implementing lua scripting here, we don't need this anymore.

File: services/ssh/ssh-simulator.go [hunk 2]
```diff
@@ -395,12 +407,28 @@ func (s *sshSimulatorService) Handle(ctx context.Context, conn net.Conn) error {
...
+								break
+							}
+
+							payload := decoder.String()
+
+							resp, _ := sConn.Handle(payload)
+							channel.Write([]byte(fmt.Sprintf("%s", resp)))
+						}
+
+						// channel.Write([]byte(fmt.Sprintf("%s: command not found\n", "ls")))
```
Comment (martijnjanssen) [inline services/ssh/ssh-simulator.go]: Same here right?

**Review by martijnjanssen — APPROVED** [review]

**Comment by martijnjanssen** [comment]
Is it wanted that this service is no longer usable as an ssh-simulator without the lua scripting?

File: services/ssh/ssh-simulator.go [hunk 3]
```diff
@@ -395,12 +407,28 @@ func (s *sshSimulatorService) Handle(ctx context.Context, conn net.Conn) error {
 								event.Custom("ssh.command", line),
 							))
 
-							term.Write([]byte(fmt.Sprintf("%s: command not found\n", line)))
+							resp, _ := sConn.Handle(line)
```
Comment (martijnjanssen) [inline services/ssh/ssh-simulator.go]: Needs error checking.

File: services/ssh/ssh-simulator.go [hunk 4]
```diff
@@ -395,12 +407,28 @@ func (s *sshSimulatorService) Handle(ctx context.Context, conn net.Conn) error {
...
+						decoder := PayloadDecoder(req.Payload)
+
+						for {
+							if decoder.Available() == 0 {
+								break
+							}
+
+							payload := decoder.String()
+
+							resp, _ := sConn.Handle(payload)
```
Comment (martijnjanssen) [inline services/ssh/ssh-simulator.go]: Error checking again

**Review by nordinvannes — APPROVED** [review]
LGTM!

**Pull request merged**